src/vm/precode.h

```diff
@@ -172,7 +172,7 @@ class Precode {
 
 #if defined(_TARGET_ARM_) && defined(HAS_COMPACT_ENTRYPOINTS)
         // Precodes have to be aligned to allow fast compact entry points check
-        _ASSERTE (align >= sizeof(void*));
+        _ASSERTE (align >= TARGET_POINTER_SIZE);
```

Comment:
This one should not be needed, I think.

Reply:
@jkotas 
I double checked and without this change x64/arm crossgen will assert compiling `System.Private.CoreLib.dll` due to the following:
1.  `align` is equal to `PRECODE_ALIGNMENT` 
2. `PRECODE_ALIGNMENT` is defined in src\vm\arm\cgencpu.h:1083 as `CODE_SIZE_ALIGN`
2. `CODE_SIZE_ALIGN` is defined in src\vm\arm\cgencpu.h:78 as `4`

Reply:
This depends on how the interesting changes are done. I am not able to tell whether this is the  change. I would leave the two change I have commented on for the review of the "interesting changes".

Reply:
@jkotas I agree, this makes sense. I will postpone the two changes for the later steps